books/free-programming-books.md

```diff
@@ -467,6 +468,17 @@
 * [Theory and Applications for Advanced Text Mining](http://www.intechopen.com/books/theory-and-applications-for-advanced-text-mining)
 
 
+#### Game Development
+
+* [2D Game Development: From Zero To Hero](https://github.com/Penaz91/2DGD_F0TH) (:construction: *in process*)
+* [Coding With Minecraft](https://turtleappstore.com/book/) - Al Sweigart
+* [Designing Virtual Worlds](https://mud.co.uk/richard/DesigningVirtualWorlds.pdf) - Richard A. Bartle (PDF)
+* [Game AI Pro](http://www.gameaipro.com) - Steve Rabin
+* [Game Design with AGS](https://ensadi.github.io/AGSBook/) - Dave Ensminger, A. G. Madi
+* [Game Programming Patterns](http://gameprogrammingpatterns.com) - Bob Nystrom
+* [Procedural Content Generation in Games](https://pcgbook.com) - Noor Shaker, Julian Togelius, Mark Nelson
```

Comment:
Browser raised `NET::ERR_CERT_AUTHORITY_INVALID`

Maybe change to `http` protocol

```suggestion
* [Procedural Content Generation in Games](http://pcgbook.com) - Noor Shaker, Julian Togelius, Mark Nelson
```